stories/link-bars/ButtonStory.js

```diff
@@ -36,14 +37,14 @@ const ButtonStory = () => (
 			<HorizontalLinkBar background={colors.white}>
 				<LinkBarButton
 					isActive
-					onClick={(e) => { alert('You like clicking things'); }} // eslint-disable-line no-alert
+					onClick={action('You like clicking things')}
```

Comment:
Doesn't it need to be {()=>action('')}?

Reply:
Ah, nevermind :)

Reply:
Not according to the action [addon's docs](https://github.com/storybooks/storybook/blob/master/addons/actions/README.md)